### ISSUE TITLE
Alerter not dismising when using enableSwipeToDismiss together with OnClickListener

### ISSUE THREAD

**Issue by FlorianWalther** [Tapadoo/Alerter#118]
Hey,
when I enable swipeToDismiss and set an OnClickListener, it doesn't dimiss after clicking and just stays there after the duration is over.

**Comment by TeTRA99** [comment]
Hi Walther, have you tried adding `Alerter.clearCurrent(getActivity());` inside the OnClickListener? It works for me..

```
.setOnClickListener(new View.OnClickListener() {
                                                    @Override
                                                    public void onClick(View view) {
                                                        Alerter.clearCurrent(getActivity());
                                                    }
                                                })
```

**Comment by kpmmmurphy** [comment]
I've added a fix for this in version 2.0.5. 

https://github.com/Tapadoo/Alerter/pull/121
